server/src/responders/user-responders.js

```diff
@@ -335,6 +336,24 @@ async function requestAccessResponder(
   await sendAccessRequestEmailToAshoat(request);
 }
 
+const friendRequestInputValidator = tShape({
+  userIDs: t.list(t.String),
+});
+
+async function friendRequestCreationResponder(
+  viewer: Viewer,
+  input: any,
+): Promise<void> {
+  const request: AddFriendsRequest = input;
+  await validateInput(viewer, friendRequestInputValidator, request);
+
+  return new Promise(resolve => {
```

Comment:
Something else we'll probably need on the server: when we launch the feature, we'll probably want to bootstrap the friend lists based on existing connections in threads. This would be a separate PR as well, but basically when we launch the friend list we can consider anybody who is in a thread with you as your friend. We probably would just set up a script in `server/src/scripts`.

I created #51 to track